src/main/java/codeu/controller/BotServlet.java

```diff
@@ -156,7 +156,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
         String responseMessageContent;
         try{
-            responseMessageContent = botService.process(cleanedMessageContent);
+            responseMessageContent = botService.process(cleanedMessageContent, user);
```

Comment:
Specify the currently logged in user as the query issuer to the BotService

Reply:
Love what you've done with this. So this allows both the ChatService and the BotService to identify the same user?

Reply:
Yup! c: